server/portal/settings/unit_test_settings.py

```diff
@@ -307,7 +307,11 @@ def gettext(s): return s
     }
 }
 HAYSTACK_ROUTERS = ['aldryn_search.router.LanguageRouter', ]
-
+"""
+SETTINGS: RECAPTCHA TESTING KEY
+"""
+RECAPTCHA_SECRET_KEY = "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
```

Comment:
can you add a comment here in code about that.

Reply:
Here is explanation about this codes.

For reCAPTCHA v2, use the following test keys. You will always get No CAPTCHA and all verification requests will pass.

Site key: 6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
Secret key: 6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe
The reCAPTCHA widget will show a warning message to ensure it's not used for production traffic.

Reply:
sorry, i wasn't so clear.  I mean like in `server/portal/settings/unit_test_settings.py`

so something like

```
"""
SETTINGS: RECAPTCHA TESTING KEY
provided by Google at https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do
"""
```

Then we'll always know that these are special values in case we forget or are a new developer,

Reply:
@nathanfranklin Updated!